omod-common/src/test/java/org/openmrs/module/webservices/rest/web/ConversionUtilTest.java

```diff
@@ -145,6 +148,26 @@ public void convert_shouldConvertToAClass() throws Exception {
 		Assert.assertTrue(converted.isAssignableFrom(String.class));
 	}
 	
+	@Test
+	public void convert_shouldConvertSimpleObjectToCustomRepresentation() throws Exception {
```

Comment:
The method name does not seem to reflect the work being done by the ticket?

Reply:
The ticket is about not being able to get custom representations with audit info. `auditInfo` is a simple object that implements map interface.

So bassically it's enough to assert that it can properly convert simple objects to a given custom representation.

Reply:
@dkayiwa Can we merge this fix?